src/modules/Todo.js

```diff
@@ -0,0 +1,68 @@
+const descriptionInput = document.getElementById('description-input');
+let allToDos = [];
+const todosList = document.getElementById('todos');
+
+const deleteToDoItem = (index) => {
+  allToDos.splice(index, 1);
+  todosList.innerHTML = '';
+  localStorage.setItem('alltasks', JSON.stringify(allToDos));
+};
+
+const editToDoList = (index, newValue) => {
```

Comment:
- [ ]  As previous reviewer suggested the functions for `editing `and `deleting `the task should work as expected. at this moment a user can not edit or delete a created task. pleasure check the requirements those functions as suggested as well.